### ISSUE TITLE
Duplicate control when add new control after delete section and then set it again

### ISSUE THREAD

**Issue by harrieanto** [sethsandaru/vue-form-builder#57]
I try your vue form builder and it's totally awesome. Thankyou. But I have found an issue when I try to set form from default template. The issue is when I delete section and then set it back from the template and then I try to add new control the Vue form builder seem create multiple/duplicate control. It would be great to fix it. Thankyou.

**Comment by sethsandaru** [comment]
Hello @harrieanto ,

Thanks for the compliment.

For the problem that you approached, it would be really helpful if you can provide me:

- The Form Configuration JSON
- More detail about how to reproduce the problem (Screenshot/A short clip would be really helpful).
- (Optional) Create a simple live demo of your application.

Thanks

**Comment by harrieanto** [comment]
Yes. Thanks for your fast response. I'll back here and provide what's needed as soon as possible.

**Comment by abhimanusharma** [comment]
@sethsandaru  I was facing similar kind of issue, so I have prepared a clip to showcase the issue `https://www.loom.com/share/87b888ed8afb41759e105d73506ac467`

**Comment by harrieanto** [comment]
@abhimanusharma exactly like that

**Comment by abhimanusharma** [comment]
@sethsandaru I think the label should be changed to bug.

**Comment by harrieanto** [comment]
@sethsandaru @abhimanusharma Hi all, Already have solusiton for the issue?

**Comment by abhimanusharma** [comment]
@harrieanto , It is still under progress, hope we will get it on new year.
